scripts/js/lib/api/saveImages.ts

```diff
@@ -44,10 +48,6 @@ export async function saveImages(
   const destFolder = pkg.outputDir(`${publicBaseFolder}/images`);
   if (!(await pathExists(destFolder))) {
     await mkdirp(destFolder);
-  } else if (pkg.isDev()) {
-    // We don't want to store images from other versions when we generate a
-    // different dev version
-    await rmFilesInFolder(destFolder);
```

Comment:
Not necessary because updateApiDocs.ts will have already removed all images as part of removing Markdown.